pkg/test/resourcefixture/testdata/basic/asset/v1beta1/assetsavedquery/assetsavedquery-minimal/create.yaml

```diff
@@ -42,4 +42,4 @@ spec:
         outputResourceEdges: true
         analyzeServiceAccountImpersonation: true
       conditionContext:
-        accessTime: "2025-08-13T10:00:00Z"
```

Comment:
https://github.com/GoogleCloudPlatform/k8s-config-connector/pull/5047 handles the case that accessTime has to be a future time (and stable in fixture test).